src/Agent/Communication/Headers/Common.php

```diff
@@ -140,7 +140,7 @@ public function getHeadersNames(): array {
     *
     * @return array
     */
-   public function getHeaders($legacy = false): array {
+   public function getHeaders($legacy = true): array {
```

Comment:
Maybe it should be set to true by default on `AbstractRequest` too.
Also, as everyone seems to pass this param to true, is it still usefull ?

Reply:
In facts, all of that is currently useless; since nothing is used... :/